extension/controller/backgroundScript.js

```diff
@@ -80,8 +81,20 @@ const messageListener = async function(message, sender) {
                      */
                     setTimeout(() => {
                         browser.tabs.sendMessage(
-                            tabId,
-                            { command: "responseMonitorTabLoad", tabId }
+                            tab.id,
+                            { command: "responseMonitorTabLoad", tabId: tab.id }
+                        );
+                    } ,250);
+                } else if (tab.id !== sender.tab.id) {
```

Comment:
This logic would activate the logic for an arbitrary tab (that happens to trigger the `tabs.onUpdated` event). Is that really what you intended?